components/wb-bubble/wb-bubble-en.html

```diff
@@ -0,0 +1,76 @@
+---
+title: Bubble Open Overlay - Provisional feature
+language: en
+category: Plugins
+description: Working example of the virtual assistant bubble.
+altLangPage: wb-bubble-fr
+dateModified: 2021-06-07
+---
+{% include alert-provisional.html siteroot="../../" %}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2021-06-07</dd>
+	<dt>Conforming to</dt>
+	<dd>n/a</dd>
+	<dt>Guidance</dt>
+	<dd>n/a</dd>
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>wb-buuble</code></li>
```

Comment:
There is a small typo here
```suggestion
			<li><code>wb-bubble</code></li>
```